src/librustdoc/core.rs

```diff
@@ -527,9 +521,11 @@ crate fn run_global_ctxt(
             .cloned()
             .filter(|trait_def_id| tcx.trait_is_auto(*trait_def_id))
             .collect(),
-        render_options,
         module_trait_cache: RefCell::new(FxHashMap::default()),
-        cache: Cache::default(),
+        cache: Cache::new(access_levels, render_options.document_private),
+        inlined: Default::default(),
```

Comment:
nit: Personally I prefer `FxHashSet::default()` (or `::new()`), since it means you don't have to scroll all the way up to figure out what it is. But not a big deal :)

```suggestion
        inlined: FxHashSet::default(),
```